### DISCUSSION TITLE
Schedule with telebot

### DISCUSSION THREAD

**Discussion by phucdc** [eternnoir/pyTelegramBotAPI#1806] [Q&A]
Hi, i want to make my bot auto-send scheduled messages (like birthday notifications), how can i do that?

**Comment by Badiboy** [comment]
Use specialized scheduling libraries. 

PS. Not an issue.

**Reply by phucdc** [reply]
do you have some example? i tried with `apschedule` and `schedule`, but seem like those won't async with other `message_handling` process

**Comment by coder2020official** [comment]
Unfortunately, this functionality is not implemented yet. I had some plans to do so.
For now, use apscheduler. They have AsyncIOScheduler - for async, and BackgroundScheduler for sync afaik.
